lib/miq_automation_engine/service_models/miq_ae_service_methods.rb

```diff
@@ -65,8 +65,7 @@ def self.vm_templates
 
     def self.category_exists?(category)
       ar_method do
-        cat = Classification.find_by_name(category)
-        cat.nil? ? false : true
+        Classification.find_by_name(category).present?
```

Comment:
CodeClimate is complaining here with "Use `find_by` instead of dynamic `find_by_name`."

I think updating our bot to use a newer rubocop would dump this as well.